RetrospectiveExtension.Frontend/scripts/ci.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+##
+## This script runs the CI tasks
+##
+(
+    cd "$(dirname "$0")/.." || exit
+    set -euo pipefail
+
+    npm install
+    echo "install dependencies"
+
+    npm run lint
+    echo "eslint passed"
+
+    npm run build:p
```

Comment:
One concern I have is that the `ci.sh` would fail locally since there is no `enviornment.tsx` file when you first clone the repo and start working. To use the pre-commit hook successfully you have to create that file. This is ok most of the times, but for cases where somebody clones the repo to update some Markdown lines or add some extra comments somewhere, the pre-commit would fail which is not a great experience. I would suggest checking if the `/config/enviornemnt.tsx` file is not found, then creating it if it is not found,

Reply:
@mhassaninmsft That's a good point. Since this file is required to build I'm wondering if we should remove the template and just check in the environment.tsx with the placeholder values to simplify. I don't think there is any downside from a dev/test perspective if the backend is not being exercised.

Reply:
It would make the development expirence worse I think, this was the case before creating the environment template. Many people including me would accedientlay check in the application inssghts instrumentation key and the url for the backend. Ideally for config file they are in Json files or loaded dynamically from the env. Here the file is actually a jsx file which is required at build time. The best solution would be to move `enviorment.tsx` into `environment` and require it at run time. I think Json files should not cause a build failure at compile time. I will verify that. Anyway I leave that final decision up to you.

Reply:
Plan is to address this in a future PR by switching to a proper config mechanism. Leaving as is for now with the known requirement that the `environment.tsx` file is copied/created prior to performing a commit.